src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,56 @@
+import { useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+import classNames from 'classnames';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sex = searchParams.get('sex') || '';
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  const centuriesArr = ['16', '17', '18', '19', '20'];
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const search = getSearchWith(searchParams, {
+      query: event.target.value || null,
+    });
+
+    setSearchParams(search);
+  };
+
+  const toggleCenturies = (item: string) => {
+    const newCenturies = centuries.includes(item)
+      ? centuries.filter(century => century !== item)
+      : [...centuries, item];
+
+    return { centuries: newCenturies };
+  };
```

Comment:
The toggleCenturies function is a bit confusing. It's not immediately obvious what its purpose is. I would recommend adding a comment to describe what it does. Also, consider renaming it to something more descriptive.